homeassistant/components/statistics/sensor.py

```diff
@@ -963,8 +958,8 @@ def _async_next_to_purge_timestamp(self) -> datetime | None:
                 _LOGGER.debug(
```

Comment:
In the future this should probably get a `logger.isEnabledFor(logging.DEBUG)` so we don't waste time formatting arguments we will throw away most of the time
https://docs.python.org/3/howto/logging.html#optimization